### PR TITLE
new(mp) : New Dell OS10 SNMP Monitoring Procedure [FR]

### PR DESCRIPTION
**Pull request by psamecentreon**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [X] 20.04.x
- [X] 20.10.x (master)

### CODE REVIEW

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
+---
+id: network-dell-os10-snmp
+title: Dell OS10 SNMP
+---
+
+## Vue d'ensemble
+
+Dell OS10 est la nouveau système d'exploitation des équipements réseau Dell. Il combine le meilleur de Linux, de l’informatique ouverte et de la mise en réseau pour faire avancer la désagrégation de mise en réseau ouverte.
+Ce Plugin-Pack, supervise tous les points physique et système.
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:9]: A supprimer

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+* Par intances :
+
+| Metric name                   | Description                               | Unit |
+|:----------------------------- |:----------------------------------------- | :--- |
+| os10CardStatus                | Status of the card                        |      |
+| os10FanOperStatus             | Status of the fan                         |      |
+| os10FanTrayOperStatus         | Status of the fantray                     |      |
+| os10PowerSupplyOperStatus     | Status of the power supply                |      |
+| resource.oper_status          | Status of the resources		    |      |
+| hardware.temperature.celsius  | temperature of the different sensors      | C    |
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:70]: cf EN review

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+| :-------------------------- | :------------ | :--- |
+| swap.usage.bytes            | Swap usage    | B    |
+| swap.free.bytes             | Swap free     | B    |
+| swap.usage.percentage       | Swap usage    | %    |
+
+<!--Uptime-->
+
+| Metric name                 | Description                                        | Unit |
+| :-------------------------- | :------------------------------------------------- | :--- |
+| uptime                      | Duration of system has been working and available. | s    |
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:103]: cf EN review bis

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+```bash
+yum install centreon-plugin-Network-Dell-Os10-Snmp.noarch
+```
+
+2. Sur l'interface Web de Centreon, installer le Plugin-Pack *Dell OS10 SNMP* depuis la page "Configuration > Plugin packs > Manager"
+
+<!--Offline IMP License-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources Dell OS10 :
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:129]: Uppercase C => Collecteurs

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+3. Sur l'interface Web de Centreon, installer le Plugin-Pack *Dell OS10 SNMP* depuis la page "Configuration > Plugin packs > Manager"
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+Ce Plugin-Pack est conçu de manière à avoir dans Centreon un hôte par équipement Dell OS10.
+Lorsque vous ajoutez un hôte à Centreon, appliquez-lui le modèle Net-Dell-Os10-SNMP-custom. 
+Il est nécessaire de remplir les valeurs des champs "SNMP Community" et "SNMP Version".
+Une fois celui-ci configuré, certaines macros doivent être renseignées:
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:150]: Cette ligne n'est pas nécessaire pour du SNMP

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+Cette commande déclenchera une alarme WARNING si l'utilisation moyenne CPU est à plus de 60% (```--warning-average='60'```)
+et une alarme CRITICAL si plus de 75% (```--critical-average='75'```).
+
+Des seuils peuvent être fixés sur toutes les métriques de l'appareil en utilisant la syntaxe "```--warning-*metric* --critical-*metric*```".
+ 
+Toutes les options qui peuvent être utilisées avec ce plugin se trouvent sur la commande ```--help``` :
+
+```bash
+/usr/lib/centreon/plugins/centreon_dell_os10_snmp.pl --plugin=network::dell::os10::snmp::plugin \
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:191]: one option / line. Use \ between script name and --plugin

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+	--mode=cpu --help
+```
+
+### UNKNOWN: SNMP GET Request : Timeout
+
+Si vous obtenez ce message, cela signifie que vous ne parvenez pas à contacter le serveur Linux sur le port 161, ou alors que la communauté SNMP configurée n'est pas correcte.
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md:197]: Not Linux Server !

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 7 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Oups!! It fixed now 😅

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 1 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Partie supprimer ✔️

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 3 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Done

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 4 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Done!

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 6 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Done ^^

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 2 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Done ^^

File: fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md (hunk 5 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md]: Done. It removed

**Review by Sims24 — APPROVED** [review]

**Pull request merged**